.github/workflows/pytest.yml

```diff
@@ -52,13 +52,13 @@ jobs:
 
       - name: run pytest
         run: |
-          pytest -n 2 --runslow tests/
+          pytest -n 2 --runslow
           echo "Exited with '$?'"
 
       - name: run pytest on notebooks (append coverage)
         if: (matrix.os == 'ubuntu-latest') && (matrix.py == '3.8')
         run: |
-          pytest -n 2 --dist loadscope --nbval --current-env --cov=weldx --cov-append tutorials/
+          pytest -n 2 --dist loadscope --nbval --current-env --cov-append tutorials/
```

Comment:
`--cov=weldx`should be set in `setup.cfg`

Reply:
I know, but the way nbval interacts with pytest-cov, it seemed mandatory to explicitly give the argument again. It shouldn't make a difference and I consider it a bug in nbval. You could try omitting it and see what happens (there will be no nb coverage).

Reply:
But eventually the cov-append flag also triggers coverage. We will see

Reply:
So the cov report is unchanged, which means I added the append flag later on during my last PR. Works like expected.